packages/shared/src/graphql/feed.ts

```diff
@@ -170,7 +173,7 @@ export const SOURCE_FEED_QUERY = gql`
       ...FeedPostConnection
     }
   }
-  ${FEED_POST_CONNECTION_FRAGMENT}
+  ${getFeedPostFragment('pinnedAt')}
```

Comment:
Why do you add them conditional? I think we should just clean them into a FeedPost no?